teams/social-media.toml

```diff
@@ -0,0 +1,9 @@
+name = "social-media"
+subteam-of = "leadership-council"
+
+[people]
+leads = ["m-ou-se"]
+members = ["m-ou-se"]
+
+[[lists]]
+address = "social-media@rust-lang.org"
```

Comment:
would a `social@rust-lang.org` alias make sense to match the URL of the mastodon server?